yb-voyager/cmd/assessMigrationCommand.go

```diff
@@ -901,18 +903,18 @@ func fetchColumnsWithUnsupportedDataTypes() ([]utils.TableColumnsDataTypes, erro
 		err := rows.Scan(&columnDataTypes.SchemaName, &columnDataTypes.TableName,
 			&columnDataTypes.ColumnName, &columnDataTypes.DataType)
 		if err != nil {
-			return nil, fmt.Errorf("error scanning rows: %w", err)
+			return nil, nil, fmt.Errorf("error scanning rows: %w", err)
 		}
 
 		allColumnsDataTypes = append(allColumnsDataTypes, columnDataTypes)
 	}
 
 	var sourceUnsupportedDataTypes []string
+	var liveMigrationUnsupportedDataTypes []string
 	switch source.DBType {
 	case POSTGRESQL:
-		sourceUnsupportedDataTypes = srcdb.PostgresUnsupportedDataTypesForDbzm
-		//adding XID here as data export import should not fail as YugabyteDB supports inserts on it but its functions are not supported
-		sourceUnsupportedDataTypes = append(sourceUnsupportedDataTypes, "xid")
+		sourceUnsupportedDataTypes = srcdb.PostgresUnsupportedDataTypes
+		liveMigrationUnsupportedDataTypes = srcdb.PostgresUnsupportedDataTypesForDbzm
```

Comment:
Can we return only the list of types unsupported in live migration (excluding the ones that are completely unsupported), we can just do that by getting the difference of these two lists for `liveMigrationUnsupportedDatatypes`? 
 this will avoid the duplication of the details in the report as we are already listing some types as unsupported which are completely not supported and then we should list only the ones not supported in live migration in the caveats section.